doc/source/conf.py

```diff
@@ -40,7 +40,6 @@
 # specify the location of your github repo
 html_theme_options = {
     "github_url": "https://github.com/ansys/ansys-sphinx-theme",
```

Comment:
```suggestion
    "github_url": "https://github.com/ansys/ansys-sphinx-theme",
    "use_edit_page_button": True,
```
I think we need this here.

Reply:
Since this is the default configuration enabled in the `theme.conf`, it is no longer required to have it in the `conf.py`.